plugins/backup/veeam/src/main/java/org/apache/cloudstack/backup/VeeamBackupProvider.java

```diff
@@ -149,7 +152,7 @@ public boolean assignVMToBackupOffering(final VirtualMachine vm, final BackupOff
         for (final BackupOffering job : client.listJobs()) {
             if (job.getName().equals(clonedJobName)) {
                 final Job clonedJob = client.listJob(job.getExternalId());
-                if (clonedJob.getScheduleConfigured() && !clonedJob.getScheduleEnabled()) {
+                if (BooleanUtils.isTrue(clonedJob.getScheduleConfigured()) && !clonedJob.getScheduleEnabled()) {
```

Comment:
why is `BooleanUtils` needed?

Reply:
Because `BooleanUtils.isTrue()` is null safe, and the method `client.listJob(job.getExternalId())` can return null.